src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
+.UserInfo {
```

Comment:
Underscores and double dashes seem to be used correctly in this file. Keep up the good work! 👍